pimd/pim_cmd.c

```diff
@@ -5769,6 +5807,60 @@ DEFUN (show_ip_multicast_vrf_all,
 	return CMD_SUCCESS;
 }
 
+DEFUN(show_ip_multicast_count,
+      show_ip_multicast_count_cmd,
+      "show ip multicast count [vrf NAME] [json]",
+      SHOW_STR IP_STR
+      "Multicast global information\n"
+      "Data packet count\n"
+      VRF_CMD_HELP_STR JSON_STR)
+{
+	int idx = 2;
```

Comment:
shouldn't be 3?

Reply:
changed to 3, will reduce the iteration by 1.  Thanks for the catch